src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.User;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
+    private User user;

```

Comment:
I think it's better to use ids instead of objects

_From checklist_:
> In your MovieSessionRequest/ResponseDtos, we don’t need to pass the whole information about Movie and CinemaHall - their ids will be enough.